### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by HarrisonMS**
# Description
Set up file structure

## Type of change
File tree structure created
- [x] This change requires a documentation update

## Change Status

- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have made corresponding changes to the documentation
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/creator/WorkoutCreator.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import header from '../notglobal/header'
```
Comment (macjabeth) [inline src/components/creator/WorkoutCreator.js]: This is importing something that doesn't exist?

File: src/setupTests.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom/extend-expect';
```
Comment (macjabeth) [inline src/setupTests.js]: Research what this is doing for you.

File: src/components/creator/WorkoutCreator.js (hunk 1 above)
Comment (HarrisonMS) [reply src/components/creator/WorkoutCreator.js]: Oh yea we were just running over an example of something forgot to delete that.

File: src/setupTests.js (hunk 2 above)
Comment (HarrisonMS) [reply src/setupTests.js]: Slightly researched looks to be testing for jest i believe was just in the create react default files. The whole file could probably be deleted based on the testing libraries we plan on using.

**Pull request merged**